symplyphysics/laws/kinematic/vector/linear_velocity_is_angular_velocity_cross_radius.py

```diff
@@ -0,0 +1,34 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    angle_type,
+    cross_cartesian_vectors,
+)
+from pytest import approx
+from symplyphysics.core.vectors.arithmetics import dot_quantity_vectors
+
+# Description
+## Assuming the body is rotating about a fixed axis, the vector of its linear velocity is the
+## cross product of the vector of angular rotation and the radius vector of the body. The radius
+## vector of the body is perpendicular to the axis of rotation.
+
+# Law: v = [w, r]
+## v - vector of linear velocity
+## w - pseudovector of angular velocity, parallel to axis of rotation
+## r - radius vector of body, perpendicular to axis of rotation
+
+
+def linear_velocity_law(angular_velocity: Vector, rotation_radius: Vector) -> Vector:
+    return cross_cartesian_vectors(angular_velocity, rotation_radius)
+
+
+@validate_input(angular_velocity_=angle_type/units.time, rotation_radius_=units.length)
+@validate_output(units.velocity)
+def calculate_linear_velocity(angular_velocity_: QuantityVector, rotation_radius_: QuantityVector) -> QuantityVector:
+    # angular_velocity_ and rotation_radius_ must be perpendicular to each other
+    assert dot_quantity_vectors(angular_velocity_, rotation_radius_).scale_factor == approx(0.0, 1e-3)
```

Comment:
Throw exception instead of assert

Reply:
I suppose we need a separate error type for that, say, `NonOrthogonalVectorsError`?

Reply:
No, just ValueError

Reply:
Okay